blobfuse/blobfuse.cpp

```diff
@@ -410,6 +418,8 @@ void *azs_init(struct fuse_conn_info * conn)
                         str_options.aadEndpoint);
             }
 
+                        fprintf(stdout , "MSI params: identityclientId %s\n", str_options.identityClientId.c_str());
```

Comment:
This will always print out, is this for debugging purposes? if anything it should be put in the syslog and under conditions of log_debug flag

Reply:
removed